internal/server/handlers_test.go

```diff
@@ -249,3 +252,113 @@ func TestDeleteIdentity_NoDeleteSelf(t *testing.T) {
 
 	assert.Equal(t, http.StatusForbidden, resp.Code, resp.Body.String())
 }
+
+func TestAPI_CreateGrant_ResponseBody(t *testing.T) {
+	srv := setupServer(t, withDefaultAdminAccessKey)
+	routes := srv.GenerateRoutes(prometheus.NewRegistry())
+
+	reqBody := strings.NewReader(`
+		{
+		  "subject": "i:12345",
+		  "privilege": "admin-role",
+		  "resource": "kubernetes.some-cluster"
+		}`)
+
+	resp := httptest.NewRecorder()
+	req, err := http.NewRequest(http.MethodPost, "/v1/grants", reqBody)
+	assert.NilError(t, err)
+	req.Header.Add("Authorization", "Bearer "+srv.options.AdminAccessKey)
+
+	accessKey, err := data.ValidateAccessKey(srv.db, srv.options.AdminAccessKey)
+	assert.NilError(t, err)
+
+	routes.ServeHTTP(resp, req)
+	assert.Equal(t, resp.Code, http.StatusCreated)
+
+	expected := jsonUnmarshal(t, fmt.Sprintf(`
+		{
+		  "id": "<any-valid-uid>",
+		  "created_by": "%[1]v",
+		  "privilege": "admin-role",
+		  "resource": "kubernetes.some-cluster",
+		  "subject": "i:12345",
+		  "created": "%[2]v",
+		  "updated": "%[2]v"
```

Comment:
We could take a different approach here, something more like `"<any-valid-uid>"`, and instead make this

```suggestion
		  "created": "<time-now>",
		  "updated": "<time-now>"
```

Which would remove the need to pass `time.Now().UTC().Format(time.RFC3339)` to `fmt.Sprintf`. 

Thoughts?

Reply:
Also, we could use a different placeholder syntax, these angle brackets are just the first thing that came to mind. We could also do:
* `"[any-valid-uid]"`,  `"[time-now]"`
* `"uid.ID(any)"`, `"time.Now()"`

These are exact string matches, so it can be anything really.